### PR TITLE
KFSPTS-6979 add missing payload element, log additional info, change access type to NONE, update date field

### PR DESCRIPTION
**Pull request by db483**

### CODE REVIEW

File: src/main/java/edu/cornell/kfs/concur/rest/resource/ConcurAccountValidationResource.java [hunk 1]
```diff
@@ -28,6 +29,7 @@ public Response validate(ConcurEventNotificationDTO concurEventNotificationDTO)
         try {
             concurEventNotification = getConcurEventNotificationConversionService().convertConcurEventNotification(concurEventNotificationDTO);
         } catch (ParseException e) {
+            LOG.error(e.getMessage());
```
Comment (jhulslander) [inline src/main/java/edu/cornell/kfs/concur/rest/resource/ConcurAccountValidationResource.java]: This should probably include the function name in the error message.  Also, should the error be logged as well as the message?

File: src/main/java/edu/cornell/kfs/concur/service/impl/ConcurEventNotificationConversionServiceImpl.java [hunk 2]
```diff
@@ -15,10 +15,13 @@
     public ConcurEventNotification convertConcurEventNotification(ConcurEventNotificationDTO concurEventNotificationDTO) throws ParseException {
         ConcurEventNotification concurEventNotification = new ConcurEventNotification();
         concurEventNotification.setContext(concurEventNotificationDTO.getContext());
-        concurEventNotification.setEventDateTime(dateTimeService.convertToSqlDate(concurEventNotificationDTO.getEventDateTime()));
+        if(concurEventNotificationDTO.getEventDateTime() != null){
```
Comment (jhulslander) [inline src/main/java/edu/cornell/kfs/concur/service/impl/ConcurEventNotificationConversionServiceImpl.java]: should this use objectUtils.isNotNull?

File: src/main/java/edu/cornell/kfs/concur/rest/resource/ConcurAccountValidationResource.java (hunk 1 above)
Comment (db483) [inline src/main/java/edu/cornell/kfs/concur/rest/resource/ConcurAccountValidationResource.java]: Good point, I will update this.

File: src/main/java/edu/cornell/kfs/concur/service/impl/ConcurEventNotificationConversionServiceImpl.java (hunk 2 above)
Comment (db483) [inline src/main/java/edu/cornell/kfs/concur/service/impl/ConcurEventNotificationConversionServiceImpl.java]: This is a field in the incoming payload xml so checking for null is enough, this is not a check on a proxied object that may or may not be materialized yet so ObjectUtils would not work here.

Comment (jhulslander) [inline src/main/java/edu/cornell/kfs/concur/service/impl/ConcurEventNotificationConversionServiceImpl.java]: OK, thanks

**Comment by db483** [comment]
Hi Jay,

I have updated the code and it is now ready for review again. Thank you!

**Comment by db483** [comment]
Hi Jay,

I have updated the code and it is now ready for review again. Thank you!

**Pull request merged**